compiler/Eta/BasicTypes/DataCon.hs

```diff
@@ -789,7 +789,7 @@ dataConUnivTyVars = dcUnivTyVars
 dataConExTyVars :: DataCon -> [TyVar]
 dataConExTyVars = dcExTyVars
 
--- | Both the universal and existentiatial type variables of the constructor
+-- | Both the universal and existential type variables of the constructor
```

Comment:
I'm not sure about this. Google suggested `existentialism` but that didn't seem right...